arc90/git-sweep
# Send errors to stderr.

# Problem description

All messages are sent to stdout.
It would be nice to have error messages to stderr. In this way we can have a cronjob which sent stdout to /dev/null but in case of errors it will sent an email.
# Changes description

Exceptions are sent to stdout.stderr
# How to try and test the changes

No test were added. Let me know if you want tests for this.

On a valid repo, run the following command and no output should be observed:
git-sweep cleanup --force > /dev/null

Outside of a repo, run the same command and the output should be observed.
